LEGO1/legoactor.cpp

```diff
@@ -49,3 +11,16 @@ LegoActor::LegoActor()
 	m_unk10 = 0;
 	m_unk74 = 0;
 }
+
+// FUNCTION: LEGO1 0x1002d210
+inline const char* LegoActor::ClassName() const
```

Comment:
Those implementations almost certainly must be in the header file, otherwise they won't be inlined and all the classes that derive from `LegoActor` and call them won't have matching `ClassName`/`IsA` functions

Reply:
Oops, I have no idea how I missed this or why I changed it. As you say it's certainly incorrect.